resources/todomvc/architecture-examples/angular/dist/index.html

```diff
@@ -1,13 +1,12 @@
-<!doctype html>
-<html lang="en">
-<head>
+<!DOCTYPE html><html lang="en"><head>
   <meta charset="utf-8">
   <title>Angular</title>
   <base href="/resources/todomvc/architecture-examples/angular/dist/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-<link rel="stylesheet" href="styles.5fd101737c0aaaef.css"></head>
+<style>@charset "utf-8";html,body{margin:0;padding:0}body{font:14px Helvetica Neue,Helvetica,Arial,sans-serif;line-height:1.4em;background:#f5f5f5;color:#111;min-width:230px;max-width:550px;margin:0 auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:300}</style><link rel="stylesheet" href="styles.5fd101737c0aaaef.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="styles.5fd101737c0aaaef.css"></noscript></head>
```

Comment:
The media=print stylesheet appears to have been restored

Reply:
removed again 😄